versions.properties

```diff
@@ -5,14 +5,14 @@
 ## Generated by $ ./gradlew refreshVersions
 ## Please, don't put extra comments in that file yet, keeping them is not supported yet.
 
-version.androidx.appcompat=1.2.0
+version.androidx.appcompat=1.3.0
```

Comment:
did you mean to update appCompat here?

Reply:
I had updated it because of a reason that I can't remember now. So I will revert it.